ush/forecast_postdet.sh

```diff
@@ -470,7 +470,7 @@ WW3_postdet() {
   while (( fhr <= FHMAX_WAV )); do
     fhr3=$(printf '%03d' "${fhr}")
     vdate=$(date --utc -d "${current_cycle:0:8} ${current_cycle:8:2} + ${fhr} hours" +%Y%m%d.%H0000)
-    ${NLN} "${COMOUT_WAVE_HISTORY}/${wavprfx}.points.f${fhr3}.bin" "${DATA}/${vdate}.out_pnt.ww3"
+    ${NLN} "${COMOUT_WAVE_HISTORY}/${wavprfx}.points.f${fhr3}.nc" "${DATA}/${vdate}.out_pnt.ww3.nc"
```

Comment:
Does this need a corressponding change https://github.com/NOAA-EMC/global-workflow/blob/6956eebebc68d1270e87284fc86decbe4fd3b008/workflow/rocoto/gfs_tasks.py#L1205?

Reply:
That line is the gridded post and the line here is the point file.  

The line that you pointed out can be changed to a log file, but I think we should wait until we change where things are written to first, which is currently not included in this PR although the underlying model updates are there for this change.

Reply:
Thank you for explaining.